orbs/phraseapp/orb.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+description: |
+  Standard orb to automatically create or merge phraseapp PRs.
+
+orbs:
+  service: "jobteaser/service@0.10.0"
+
+executors:
+  ci:
+    docker:
+      - image: "jobteaser/circleci-phraseapp-phraseapp:0.0.0"
```

Comment:
**question**

Is the version correct ? Shouldn't we at least use a `0.0.1` version?